sympy/stats/crv_types.py

```diff
@@ -516,7 +516,7 @@ def BetaNoncentral(name, alpha, beta, lamda):
 
     alpha : Real number, `\alpha > 0`, a shape
     beta : Real number, `\beta > 0`, a shape
-    lamda: Real number, `\lambda \ge 0`, noncentrality parameter
+    lamda : Real number, `\lambda \geq 0`, noncentrality parameter
```

Comment:
\ge and \geq are the same. So any goes, but no need to modify and more of these.

Reply:
What could be the possible reason that on the documentation in `BetaNonCentral` parameter it shows "lamda: Real number, `lambda >= 0`, noncentrality parameter"
And not the `symbol` for lambda?

After updating this with `geq` I was able to see `symbol` for lambda.

Reply:
OK! I wouldn't expect that to be the reason (more that the space enables equations in the description, without a space everything will be the "parameter name" where one maybe cannot have equations).

If the official docmentation shows that, it can be that I had a similar PR (although clearly still a bit sloppy) that changed it to \ge some time ago, that probably didn't make it into 1.10. If you look at the dev-documentation it says `lambda ge 0` https://docs.sympy.org/dev/modules/stats.html?highlight=betanoncentral#sympy.stats.BetaNoncentral (which is caused by the lack of space before :).

Reply:
If you can add an empty line before line 555 that would also be nice. (I cannot mark there directly, but that line does not show up as code.)

Reply:
I don't know much about the `dev-documentation` is it a bit different like, are there some changes which are different from here?

Reply:
The dev documentation is for the head of the master branch. So when modifications are made it will only be shown in dev until a release is made. (Right now 1.10 is the "official" documentation, your changes here will show up in 1.11, but until then one can see them in the dev documentation.)

Reply:
Thank you : )
Now, I know the difference clearly.